--- /home/andrew/Github/AlexaPi/setup.sh	2017-02-14 13:01:33.083942183 -0800
+++ /home/andrew/Github/AlexaPi2/setup.sh	2017-02-14 17:05:58.437896286 -0800
@@ -1,12 +1,6 @@
-#! /bin/bash
+#! /bin/sh
 
-apt-get update
-apt-get install libasound2-dev memcached mpg123 python-alsaaudio -y
-easy_install pip
-pip install -r requirements.txt
-cp initd_alexa.sh /etc/init.d/AlexaPi
-update-rc.d AlexaPi defaults
-touch /var/log/alexa.log
+echo "" > creds.py
 
 echo "Enter your Device Type ID :"
 read productid
@@ -28,9 +22,7 @@
 read secret
 echo Client_Secret = \"$secret\" >> creds.py
 
-ip=`hostname -I`
-python ./auth_web.py 
-echo "Open http://$ip:5000"
+python ./auth_web.py
 
 echo "You can now reboot"
 
